runtime/oti/j9nonbuilder.h

```diff
@@ -2124,6 +2126,7 @@ typedef struct J9ConstantPool {
 #define J9CPTYPE_METHOD_TYPE  13
 #define J9CPTYPE_METHODHANDLE  14
 #define J9CPTYPE_ANNOTATION_UTF8  15
+#define J9CPTYPE_CONSTANT_DYNAMIC 18
```

Comment:
17 or 18?  The numbering here and above should agree

Reply:
J9CPTYPE is our ROM class tag which doesn't match classfile CP due to optimizations (split entries for methodref and others)

Reply:
Tag 16, 17 are used for the sharedCPEntry consistency check.
I'll move this definition down so the these appear in order

Reply:
It would be better to move `J9CPTYPE_INTERFACE_INSTANCE_METHOD` and keep `J9CPTYPE_CONSTANT_DYNAMIC` as 17.  Its clearer when we can be consistent between our internal numbers and the ones in the spec

Reply:
I'd like to keep `J9CPTYPE_INTERFACE_STATIC_METHOD & J9CPTYPE_INTERFACE_INSTANCE_METHOD` continuous as they were used together.
I'll move ConstantDynamic to 17 and static/instance to 18,19 respectively, would that be okay?

Reply:
Fine by me.  Don't forget to update the DDR / cfdump code when moving the constants